src/util/transactionUtils.ts

```diff
@@ -14,8 +14,39 @@
 // You should have received a copy of the GNU General Public License
 // along with Parity Bridges UI.  If not, see <http://www.gnu.org/licenses/>.
 
-import { TransactionStatusType, TransactionStatusEnum } from '../types/transactionTypes';
+import {
+  TransactionStatusType,
+  TransactionStatusEnum,
+  Payload,
+  TransactionDisplayPayload
+} from '../types/transactionTypes';
 
 export function isTransactionCompleted(transaction: TransactionStatusType): boolean {
   return transaction.status === TransactionStatusEnum.COMPLETED;
 }
+
+interface ToObject {
+  [propName: number]: number;
+}
+
+function toObject(arr: Array<any>): Object {
+  const rv: ToObject = {};
+  for (let i = 0; i < arr.length; ++i) rv[i] = arr[i];
+  return rv;
+}
```

Comment:
```suggestion
function toObject(arr: Array<any>): Object {
 return Object.fromEntries(Object.entries(arr));
}
```

I'd suggest to make it internal to `getTransactionDisplayPayload` - with shortr implementation we can even inline this.